civic_europe/assets/scss/core/_buttons.scss

```diff
@@ -1,3 +1,55 @@
+@mixin button-variant($color, $background, $border) {
+    color: $color;
+    background-color: $background;
+    border-color: $border;
+
+    &.focus,
+    &:focus {
+        color: $color;
+        background-color: darken($background, 10%);
+        border-color: darken($border, 25%);
+    }
+
+    &:hover {
+        color: $color;
+        background-color: darken($background, 10%);
+        border-color: darken($border, 12%);
+    }
+
+    &.active,
+    &:active,
+    .open > &.dropdown-toggle {
+        color: $color;
+        background-color: darken($background, 10%);
+        background-image: none;
+        border-color: darken($border, 12%);
+
+        &.focus,
+        &:focus,
+        &:hover {
+            color: $color;
+            background-color: darken($background, 17%);
+            border-color: darken($border, 25%);
+        }
+    }
+
+    &.disabled,
+    &[disabled],
+    fieldset[disabled] & {
+        &.focus,
+        &:focus,
+        &:hover {
+            background-color: $background;
+            border-color: $border;
+        }
+    }
+
+    .badge {
+        color: $background;
+        background-color: $color;
+    }
+}
+
```

Comment:
I'm not sure why the mixins doesn't load but I tried a bunch of stuff before just adding it here as we do this in a+, I will add an issue to find a solution with loading correctly in bootstrap